src/index.js

```diff
@@ -228,6 +218,7 @@ function bindRouteSet (routes = [], controller, server, options = {}) {
     const routePath = path.posix.join(routePrefix, route.path)
     route.methods.forEach(method => {
       try {
+        console.log(`${routePath} ${method.toUpperCase()}`)
```

Comment:
stray console.log

Reply:
Added purposely in order to print routes in the same fashion as is done for plugin routes.